include/glow/Base/Tensor.h

```diff
@@ -243,6 +243,9 @@ class Tensor final {
 
     if (size()) {
       size_t count = size() * type_.getElementSize();
+      // We are allocating memory specifically for this tensor,
+      // thus, it owns it.
+      isUnowned_ = false;
```

Comment:
can you reproduce this in a test so that we have sanitizer to complain about this issue?

Reply:
Added a test case in the updated PR.